res/css/_common.pcss

```diff
@@ -471,11 +471,10 @@ legend {
 }
 
 @define-mixin customisedCancelButton {
-    mask: url("$(res)/img/cancel.svg");
-    mask-repeat: no-repeat;
-    mask-position: center;
-    mask-size: cover;
-    background-color: $dialog-close-fg-color;
+    background-image: url("$(res)/img/cancel.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
     cursor: pointer;
     position: unset;
```

Comment:
I don't understand this change. Why is it needed, and I don't think the SVG will be tinted when changing the theme to dark or light.

Could you please revert this?

Reply:
**Background color** also won't tined with dark or light theme because It looks same for two themes, Manipulating the background color and using as cancel icon which is not recommend

Instead using Cancel button svg which is easily maintainable.

Reply:
This does not need to be changed, so i would request to either revert this as part of this pull request or doing it as described in https://github.com/matrix-org/matrix-react-sdk/blob/develop/docs/icons.md